az-hdfs-viewer/src/main/java/azkaban/viewer/hdfs/AvroFileViewer.java

```diff
@@ -0,0 +1,196 @@
+/*
+ * Copyright 2012 LinkedIn Corp.
```

Comment:
update copy right

Reply:
This is old code. Let's change the copyright when we do refactors or format changes to these legacy codes.